src/instructlab/model/backends/backends.py

```diff
@@ -234,16 +243,10 @@ def ensure_server(
     """Checks if server is running, if not starts one as a subprocess. Returns the server process
     and the URL where it's available."""
 
-    try:
-        logger.info(f"Trying to connect to model server at {api_base}")
-        # pylint: disable=duplicate-code
-        list_models(
-            api_base=api_base,
-            http_client=http_client,
-        )
+    # pylint: disable=no-else-return
+    if check_api_base(api_base, http_client):
         return (None, None, api_base)
-        # pylint: enable=duplicate-code
-    except ClientException:
+    else:
```

Comment:
any reason you disabled the lint error here instead of removing the else statement?

Reply:
It was counterproductive. Removing `else` statement caused changes in indentation, much longer diff, continuous rebases and difficulties with reviews and absence of approval.
I propose merger this PR, and after that remove the `else` with indentation.
It is more practical.